packages/shared/src/components/MainLayout.tsx

```diff
@@ -101,28 +90,7 @@ function MainLayoutComponent({
   useAuthErrors();
   useAuthVerificationRecovery();
   useNotificationParams();
-  useReferralReminder();
-  useStreakMilestone();
-
-  const marketingCta = getMarketingCta(MarketingCtaVariant.Popover);
-
-  useEffect(() => {
-    if (marketingCta) {
-      openModal({
-        type: LazyModal.MarketingCta,
-        props: {
-          marketingCta,
-          onAfterOpen: () => {
-            trackEvent({
-              event_name: AnalyticsEvent.Impression,
-              target_type: TargetType.MarketingCtaPopover,
-              target_id: marketingCta.campaignId,
-            });
-          },
-        },
-      });
-    }
-  }, [marketingCta, openModal, trackEvent]);
+  useBootPopups();
```

Comment:
As we talked this should be a component we render to avoid re-renders due to `useBootPopups` dependencies, for example in place where `ReputationPrivilegesModalTrigger` was.

Reply:
What would we render as there is no UI?

Reply:
Just return null from component.